packages/react/src/components/AccountSettings/AccountSettings.tsx

```diff
@@ -0,0 +1,4 @@
+import { ChangePassword } from './ChangePassword';
+import { DeleteUser } from './DeleteUser';
+
+export const AccountSettings = { ChangePassword, DeleteUser };
```

Comment:
Can we make this a default export since the namespace matches the file name?

Reply:
Done in [473d053](https://github.com/aws-amplify/amplify-ui/pull/3118/commits/473d0533abaecaf85a23b666fde6742eb00bcd72)

Reply:
Plus export fix in [e17c273](https://github.com/aws-amplify/amplify-ui/pull/3118/commits/e17c273cbab932c93842c2af881a638530c0dfb4)